server/routes/auth/arena.js

```diff
@@ -0,0 +1,48 @@
+const router = require('express').Router()
+const request = require('request')
+const { ARENA_CLIENT_ID, ARENA_CLIENT_SECRET } = require('../../../secrets')
+const { User } = require('../../db')
+module.exports = router
+
+router.get('/callback', function(req, res) {
+  if (!req.user || !req.user._id) return res.status(400).send('No user')
+
+  // we'll use this check for the deployed version
+  // if (!req.query.code) return res.status(400).send('No code')
+
+  const url =
+    'https://dev.are.na/oauth/token?client_id=' +
+    ARENA_CLIENT_ID +
+    '&client_secret=' +
+    ARENA_CLIENT_SECRET +
+    '&code=' +
+    '<auth code received after authenticating on frontend>' +
```

Comment:
This is where you'll put you're auth code that you'll get after authenticating with are.na on the frontend.